go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	github.com/prometheus/client_golang v1.16.0
 	github.com/sirupsen/logrus v1.9.0
 	github.com/stretchr/testify v1.8.2
-	github.com/vmware-tanzu/nsx-operator/pkg/apis v1.0.0
```

Comment:
I think the replace in line #5 and #6 could be removed.

Reply:
+1

Reply:
for nsx-operator build, do you want to use the local file from pkg/apis and pkg/client folder or a specific version?

Reply:
v0.0.1 would be good enough.

Reply:
change to use the local files for pkg/apis and pkg/client instead of tags, to avoid changes when tags change.